ddsrouter_core/test/unittest/types/endpoint/EndpointTest.cpp

```diff
@@ -95,6 +95,20 @@ Guid random_valid_guid(
         seed);
 }
 
+/**
+ * Test \c EndpointKind string conversions
+ */
+TEST(EndpointTest, string_conversions)
+{
+    ASSERT_EQ(std::string(EndpointKindStrings[static_cast<EndpointKindType>(EndpointKind::invalid)]),
```

Comment:
Is this the way the `to_string` method for this enum will be used? I would use a specific funcion: the one we will use on our code.